src/app/pages/interest-form/interest-form.hbs

```diff
@@ -0,0 +1,201 @@
+<h1>OpenStax College Interest Form</h1>
+<p>
+    Thank you for your interest in OpenStax College textbooks! Please complete the form below so that we may provide
+    you with the information you need to start saving your students thousands of dollars each semester.
+</p>
+<form accept-charset="UTF-8" action="/adoption_interests" class="new_adoption_interest" method="post">
+    <div>
+        <input name="utf8" type="hidden" value="✓">
+        <input name="authenticity_token" type="hidden" value="Xi14qRzCbSy2zkso1VEjKU4bEUfFWOrhWbQnh7tlO3c=">
```

Comment:
Looking into further, I think you could just remove this entire `<div>` and both hidden values (since we aren't getting a CSRF token from the server at the moment).  The utf8 hidden value exists to trick old versions of IE into properly submitting the form as UTF-8, since it doesn't respect the defined charset, although that issue isn't relevant on any browsers we support.